.github/workflows/pull-request.yml

```
@@ -0,0 +1,666 @@
+      - name: Apply Database Migrations
+        run: |
+          docker exec talawa-api-1 /bin/bash -c 'pnpm apply_drizzle_migrations'
+
+      - name: Start Backend Server
+        run: |
+          docker exec -d talawa-api-1 /bin/bash -c 'pnpm run start_development_server'
+          sleep 15
+
+      - name: Wait for GraphQL endpoint to become available
+        if: success()
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2034:warning:2:1: i appears unused. Verify use (or export if used externally)

(shellcheck)